orttraining/orttraining/core/optimizer/compute_optimizer/padding_elimination.cc

```diff
@@ -309,7 +309,8 @@ void IterateSubgraphFromNode(Graph& graph,
         continue;
       }
       auto func_name = static_cast<std::string>(cur->GetAttributes().at("name").s());
-      if (func_name == "_InspectActivation" || func_name == "_IncrementStep") {
+      if (func_name == "onnxruntime.training.utils.hooks._subscriber_manager._InspectActivation" ||
+          func_name == "onnxruntime.training.utils.hooks._subscriber_manager._IncrementStep") {
```

Comment:
```suggestion
      if (func_name in ["onnxruntime.training.utils.hooks._subscriber_manager._InspectActivation", "onnxruntime.training.utils.hooks._subscriber_manager._IncrementStep"]) {
```

Reply:
I believe you thought it is orttraining/orttraining/core/optimizer/compute_optimizer/padding_elimination.py. :)